app/build.gradle

```diff
@@ -38,11 +38,11 @@ android {
     buildTypes {
         debug {
             testCoverageEnabled true
-            minifyEnabled false
+            minifyEnabled true
```

Comment:
don't enable on debug, that's just to test if it works on not

Reply:
Done